crates/persist/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "bdk_persist"
 homepage = "https://bitcoindevkit.org"
-version = "0.1.0"
+version = "0.2.0"
```

🔧 Don't need to bump this one since it's the first release. 
```suggestion
version = "0.1.0"
```